lib/Pipeline/Loader.cpp

```
@@ -22,6 +22,13 @@ llvm::Error Loader::parseStepDeclaration(Runner &Runner,
                                          std::string &LastAddedStep) const {
   auto &JustAdded = Runner.emplaceStep(LastAddedStep, Declaration.Name);
   LastAddedStep = Declaration.Name;
+
+  if (Declaration.Artifacts.isValid()) {
+    auto &KindName = Declaration.Artifacts.Kind;
+    const Kind *Kind = Runner.getKindsRegistry().find(KindName);
```

What if it's not there?
We need to report an error gracefully.